Workbooks/HDInsight/At-Scale/settings.json

```diff
@@ -1,6 +1,6 @@
 {
     "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
-    "name":"HDInsight At-Scale Insights",
+    "name":"HDInsight Clusters",
```

Comment:
As of a recent change, the settings.json file is now obsolete. Sync your branch with master and there should be a gallery/hdicluster-insights/Azure Monitor and gallery/workbook/Azure Monitor. Change the new name of the workbook in those files